resolve/matrix/MatrixHandlerCuda.cpp

```diff
@@ -317,19 +325,27 @@ namespace ReSolve {
                                 CUSPARSE_ACTION_NUMERIC,
                                 CUSPARSE_INDEX_BASE_ZERO,
                                 CUSPARSE_CSR2CSC_ALG1,
-                                d_work);
+                                buffer_transpose);
     error_sum += status;
-    if (status) {
-      out::error() << "Transpose status: "   << status                    << ". "
-                   << "Last error code: " << mem_.getLastDeviceError() << ".\n";
-    }
-    mem_.deleteOnDevice(d_work);
-
     // Values on the device are updated now -- mark them as such!
     At->setUpdated(memory::DEVICE);
 
     return error_sum;
-
   }
 
+  /**
+   * @brief Add a constant to all nonzero values in the matrix
+   *
+   * @param[in, out] A - matrix
+   * @param[in] alpha - constant to be added
+   *
+   * @return int error code, 0 if successful
+   */
+  int MatrixHandlerCuda::addConstantToNonzeroValues(matrix::Sparse* A, real_type alpha)
+  {
```

Comment:
How about shortening this name?
```suggestion
  int MatrixHandlerCuda::addConstantToNonzeros(matrix::Sparse* A, real_type alpha)
  {
```